From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Fredric Silberberg <fredric@fsilberberg.com>
Date: Mon, 3 Mar 2014 11:30:55 -0500
Subject: [PATCH 0025/6262] .gitreview file

Added in .gitreview file that allows git review -s to autoconfigure
the gerrit remote, prompting for a username.

Change-Id: I59752c2ec8921b4cfb8532fceed8ef25a6682909
---
 .gitreview | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 .gitreview

diff --git a/.gitreview b/.gitreview
new file mode 100644
index 0000000000000000000000000000000000000000..f115ac15aa659a1119c44586285243afccc53f6c
--- /dev/null
+++ b/.gitreview
@@ -0,0 +1,5 @@
+[gerrit]
+host=usfirst.collab.net
+port=29418
+project=allwpilib
+defaultbranch=master
\ No newline at end of file
